IDEAS/Examples/Tutorial/DetailedHouse/DetailedHouse7.mo

```diff
@@ -0,0 +1,85 @@
+  Modelica.Blocks.Logical.Hysteresis hys(uLow=273.15 + 40, uHigh=273.15 + 45)
+    "Hysteresis controller"
+    annotation (Placement(transformation(extent={{60,-80},{80,-60}})));
+  Modelica.Blocks.Math.BooleanToReal booToRea(realTrue=0, realFalse=1)
+    "Conversion to real control signal"
+    annotation (Placement(transformation(extent={{100,-80},{120,-60}})));
+
+equation
+  connect(hys.y, booToRea.u) annotation (Line(points={{81,-70},{88,-70},{88,-70},
+          {98,-70}}, color={255,0,255}));
+  connect(booToRea.y, heaPum.y)
+    annotation (Line(points={{121,-70},{187,-70},{187,-2}}, color={0,0,127}));
+  connect(senTemSup.T, hys.u) annotation (Line(points={{136,49},{136,4},{120,4},
+          {120,-40},{40,-40},{40,-70},{58,-70}}, color={0,0,127}));
+  annotation (
+    Documentation(revisions="<html>
+<ul>
+<li>
+September 18, 2019 by Filip Jorissen:<br/>
+First implementation for the IDEAS crash course.
+</li>
+</ul>
+</html>", info="<html>
+<p>
+This step adds a controller that disables the heat pump when the supply water
+temperature exceeds <i>45°C</i>. The simple controller has a large impact on the heat pump COP.
+</p>
+<h4>Required models</h4>
+<ul>
+<li>
+<a href=\"modelica://IDEAS.Fluid.Sensors.TemperatureTwoPort\">
+IDEAS.Fluid.Sensors.TemperatureTwoPort</a>
+</li>
+<li>
+<a href=\"modelica://Modelica.Blocks.Logical.Hysteresis\">
+Modelica.Blocks.Logical.Hysteresis</a>
+</li>
+<li>
+<a href=\"modelica://Modelica.Blocks.Math.BooleanToReal\">
+Modelica.Blocks.Math.BooleanToReal</a>
+</li>
+</ul>
+<h4>Connection instructions</h4>
+<p>
+The temperature sensor between the storage tank and the secondary circulation pump serves as an input to the hysteresis controller.
+The controller is configured such that it switches to a <i>false</i> signal below <i>40°C</i> and to <i>true</i> above <i>45°C</i>.
+</p>
+<p>
+The output of the hysteresis controller is thus true when the supply temperature is high enough and false
+otherwise. This Boolean signal has to be converted in a real control signal that can be accepted by the heat
+pump model using the <code>BooleanToReal</code> block. The heat pump already has a control signal.
+Since blocks cannot be removed from an extension of a model, the heat pump model input type is set to 
+enable_variable_speed=true. This configuration allows the model to accept any real signal while ignoring 
+connections to the other control signal.
+</p>
+<h4>Reference result</h4>
+<p>
+The figure below compares the results with the results without controller of <a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse6\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse6</a>. We see that indeed,
+the supply temperature is reduced significantly. This causes the zone temperature to be slightly lower, up to
+about <i>0.25°C</i>. The COP however increases from about <i>2.9</i> to about <i>3.1</i>. Consequently, the energy use over the
```

Comment:
The increase in COP is relatively small. @lucasverleyen do we still want to change this (see https://github.com/open-ideas/__CrashCourse__/issues/19)?
The energy use reduces by +/- 2 kWh (about 15%), which is significant I would say.

Reply:
I will change the documentation such that it focuses on the electricity use